front/app/containers/Admin/settings/customize/index.tsx

```diff
@@ -255,26 +246,24 @@ class SettingsCustomizeTab extends PureComponent<
     const { locale, tenant } = this.state;
 
     if (!isNilOrError(locale) && !isNilOrError(tenant)) {
-      const homepageInfoPage = tenant.data.attributes.homepage_info_multiloc;
-
       const {
         logo,
-        header_bg,
+        // header_bg,
         attributesDiff,
         logoError,
-        headerError,
-        titleError,
-        subtitleError,
+        // headerError,
```

Comment:
Just commenting these out for now, I think we should add an additional task to clean up this file once we've moved all the flexible pages functionality out and it's clear what can go

Reply:
That file will still be used for some important settings, so I think cleaning it up in a specific task rather than over the course of 5 PRs would be safer. I  can do it here or make a follow-up ticket, either way